TOC.yml

```diff
@@ -0,0 +1,4 @@
+Technical oversight committe
+ - Ankit Mohaptra 
```

Comment:
Let's make it alphabetical and add affiliation.
So I am from dexai robotics

Reply:
Yes, lets do this properly, my attempt here was just to add a page, really. Needs lots more work :)